### ISSUE TITLE
1.14.4 error

### ISSUE THREAD

**Issue by LimboBoy2000** [maruohon/litematica#137]
I dont know what happened but it doesnt look good
![image](https://user-images.githubusercontent.com/63385156/78855912-090e8780-79da-11ea-85e3-85cb3af75f8f.png)

**Comment by SunSerega** [comment]
https://www.curseforge.com/minecraft/mc-mods/litematica

> NOTE: If you have Optifine installed, then there are some options that you most likely will need to disable in Optifine, or otherwise it will break the schematic rendering:
>
> - Render Regions needs to be disabled
> - Shaders need to be set to OFF, or alternatively disable VBOs in the vanilla Video settings, and disable both the 'schematicOverlayModelOutline' and 'schematicOverlayModelSides' options in the Visuals category
> - Fast Render might also need to be disabled if there are still some weird HUD/GUI rendering issues

Have you already read and tried that?

**Comment by vacla** [comment]
Not sure if this works for 1.14.x but in 1.15.2 apparently if you use shaders, you can load up the game with shaders and still use them

**Comment by maruohon** [comment]
Yep this is caused by the changed vertex format when using shaders. I can potentially (partially?) fix this though.
